weave-js/src/react.tsx

```diff
@@ -309,13 +309,13 @@ export const useNodeValue = <T extends Type>(
         throw new Error('client not initialized!');
       }
       if (callSite != null) {
-        console.log('useNodeValue subscribe', callSite, node);
+        // console.log('useNodeValue subscribe', callSite, node);
```

Comment:
commented out console logs -- did you want to keep them?

Reply:
I tend to opt for commenting out people's logs instead of removing in case they are useful for debugging by the original author. Especially when they look like some extra effort went into them